inc/cleantalk-settings.php

```diff
@@ -1128,12 +1129,12 @@ function apbct_settings__display()
         echo ! empty($group['html_after']) ? $group['html_after'] : '';
     }
 
-    echo '<br>';
-    echo '<div id="apbct_settings__before_button_section"></div>';
+    echo '<div id="apbct_settings__after_advanced_settings"></div>';
     echo '<div id="apbct_settings__button_section"><button name="submit" class="cleantalk_link cleantalk_link-manual" value="save_changes">'
          . __('Save Changes')
          . '</button></div>';
 
+    echo '<div class="clear"></div>';
```

Comment:
maybe this is an extra element and you can add the result with styles?
Usually clear is added if it is made up via float

Reply:
Fixed https://github.com/CleanTalk/wordpress-antispam/pull/52/commits/6cd8337ff5a3257a43d4adfd9f6fae80588720f6